uaclient/contract.py

```diff
@@ -228,3 +244,41 @@ def redact_sensitive(contract_response):
         else:
             redacted[key] = value
     return redacted
+
+
+def request_contract_updates(cfg, contract_token):
+    """Request contract updates from ua-contracts service.
+
+    @param cfg: Instance of UAConfig for this machine.
+    @param contract_token: Token of the contract we are updating.
+
+    @return: True on success False otherwise.
+    """
+    contract_client = UAContractClient(cfg)
+    if not contract_token:  # Crawl the accounts contract routes to get details
```

Comment:
As written, the function signature and documentation don't suggest `contract_token` can be False-ish; can we be more explicit about that?

Reply:
I think we now validate contract_token falseish and exit 1